docker/build.sh

```diff
@@ -6,33 +6,40 @@ set -euo pipefail
 printf "⭐️ Setting up spack environment for Dyninst\n"
 . /opt/spack/share/spack/setup-env.sh
 spack env activate .
-mkdir -p build/dyninst
 
 # 1. Build Dyninst
 printf "⭐️ Preparing to build Dyninst\n"
-echo "::group::build dyninst"   
-cd build/dyninst
-cmake -H/code -B. -DCMAKE_INSTALL_PREFIX=. > >(tee config.out) 2> >(tee config.err >&2)
-make VERBOSE=1 -j2 > >(tee build.out) 2> >(tee build.err >&2)
-make install VERBOSE=1 -j2 > >(tee build-install.out) 2> >(tee build-install.err >&2)
+echo "::group::build dyninst"
+
+DYNINST_BUILD_DIR=/opt/dyninst-env/build/dyninst
+mkdir -p $DYNINST_BUILD_DIR
+
+DYNINST_INSTALL_DIR=/opt/dyninst-env/install/dyninst
+mkdir -p $DYNINST_INSTALL_DIR
+
+cmake -S /code -B $DYNINST_BUILD_DIR -DCMAKE_INSTALL_PREFIX=$DYNINST_INSTALL_DIR
```

Comment:
You'll probably want to make groups around here (see the part you deleted below) so the output is meaningfully organized in the action.

Reply:
Sorry, I don't follow. The beginning of the dyninst build group is on line 12 and ends on 24.

Reply:
Sorry I’m just blind didn’t see it. :)

Reply:
No worries. I need someone to double-check! I'm still feeling my way through this.